exercises/farewell-workflow/README.md

```diff
@@ -0,0 +1,59 @@
+# Temporal 101 Farewell Exercise
+
+# Exercise 3: Farewell Workflow
+During this exercise, you will create an Activity function,
+register it with the Worker, and modify a Workflow to execute it.
+
+Before continuing, press Ctrl-C to stop any Worker instances still running from any
+previous exercises.
+
+As with other exercises, you should make your changes in the `practice` 
+directory. Look for "TODO" comments, which will provide hints about what
+you'll need to change. If you get stuck and need additional hints, or 
+if you want to check your work, look at the completed example in the
+`solution` directory. 
+
+## Part A: Write an Activity Function
+The `TranslateActivities.cs` file contains a method(`GetSpanishGreeting`) that uses a 
+microservice to get a customized greeting in Spanish. Follow these steps to create 
+a new Activity that gets a custom farewell message from the microservice:
+
+1. Open the `TranslateActivities.cs` file. This is located in the `practice/Workflow`
```

Comment:
Just give them the full path. The following sentence makes them have to construct the path in their head
```suggestion
1. Open the `TranslateActivities.cs` file. This is located in the `exercises/farewell-workflow/practice/Workflow`
```